src/auth/auth.action.js

```diff
@@ -1,10 +1,17 @@
+import { AsyncStorage } from 'react-native';
 import {
   fetchAccessToken,
   fetchAuthUser,
   fetchAuthUserOrgs,
   fetchUserEvents,
 } from 'api';
-import { LOGIN, GET_AUTH_USER, GET_AUTH_ORGS, GET_EVENTS } from './auth.type';
+import {
+  LOGIN,
+  LOGOUT,
+  GET_AUTH_USER,
+  GET_AUTH_ORGS,
+  GET_EVENTS,
+} from './auth.type';
```

Comment:
Don't think we really need to here since it's calling it within the same directory right? Or do you still prefer having global imports here as well?